### DISCUSSION TITLE
Can't upload enrollment transcript in education to proof academic status

### DISCUSSION THREAD

**Discussion by Maamue** [community/community#17780] [GitHub Education]
Hi everyone, 

I know this has been asked countless of times, but _maybe_ additional posts may finally somehow get the ball rolling. I have the luxury to at least see the "take a picture" button, but I am enrolled in a remote/distance teaching university and so all my documents and transcripts are files I can download only from their portal, they are not sent physically (environmental friendliness and savings of cost). It is impossible for me to upload the necessary files because there is no button or option to uload documents from my computer. I do not have a printer at home (who does nowadays?!) and thus can't print the transcript either. Please please fix the page and allow us to upload documents from our workstation without forcing us to take a picture. I tried Windows Firefox and Chrome, Debian Firefox and Chrome, and Android Firefox and Chrome, and neither of them work. 

Kind regards

Maamue

Edit: Apparently the document with the costs for the current semester (only thing i get sent physically) was enough (probably because the mail was verified?), but please please do fix this issue and allow upload from workstations. This is driving students all over the world crazy.

**Comment by mlarasusan** [comment]
Happy to know it's solved and tks for sharing your feedback and experience :]

**Comment by sharklayer** [comment]
I'm facing almost the same issue. I don't have a camera to take a picture on my PC and don't have a printer to print the transcript, so can't upload any proof of my academic status
